osier/technology.py

```diff
@@ -209,7 +230,7 @@ def unit_time(self, value):
 
     @property
```

Comment:
Allows access to this class parameter as an attribute rather than a method.
```py
print(my_obj.name)
# "Nathan Ryan"
# without @property
print(my_obj.name())
# "Nathan Ryan"
```
[This video explains a lot](https://www.youtube.com/watch?v=jCzT9XFZ5bw)